ext/web/03_abort_signal.js

```diff
@@ -25,6 +27,8 @@
   const illegalConstructorKey = Symbol("illegalConstructorKey");
 
   class AbortSignal extends EventTarget {
+    #timerId = null;
```

Comment:
this should be a symbol, not private prop

Reply:
Done